test/operation-hooks.suite/embeds-one-create.suite.js

```diff
@@ -0,0 +1,186 @@
+// Copyright IBM Corp. 2015,2016. All Rights Reserved.
+// Node module: loopback-datasource-juggler
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+var ValidationError = require('../..').ValidationError;
+
+var contextTestHelpers = require('../helpers/context-test-helpers');
+var ContextRecorder = contextTestHelpers.ContextRecorder;
+var aCtxForModel = contextTestHelpers.aCtxForModel;
+
+var uid = require('../helpers/uid-generator');
+var HookMonitor = require('../helpers/hook-monitor');
```

Comment:
>we don't have any hard rule for ordering and grouping variables (module imports)
Oh I see... yea it was just a nitpick...I just saw some Strongloop folks prefer to have ASCI order of variables...
Something like this:
```
var ContextRecorder = contextTestHelpers.ContextRecorder;
var HookMonitor = require('../helpers/hook-monitor');
var ValidationError = require('../..').ValidationError;
var aCtxForModel = contextTestHelpers.aCtxForModel;
var contextTestHelpers = require('../helpers/context-test-helpers');
var uid = require('../helpers/uid-generator');
```